api/pkg/service/ok/ok_test.go

```diff
@@ -0,0 +1,20 @@
+package ok
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/tektoncd/hub/api/pkg/testutils"
+)
+
+func TestOk(t *testing.T) {
```

Comment:
I think we can skip this test, Only HTTP test would be enough to test status code and body..